### PR TITLE
[pwa] add a handful of /team/ page ui tests

### PR DESCRIPTION
**Pull request by tervay**

### CODE REVIEW

File: pwa/tests/team.spec.ts [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+    ).toBeVisible();
+    await expect(
+      page.locator('li').filter({ hasText: 'Monterey Bay' }),
+    ).toBeVisible();
+    await expect(
+      page.locator('li').filter({ hasText: 'Milstein' }),
+    ).toBeVisible();
+    await expect(
+      page.locator('li').filter({ hasText: 'Einstein' }),
+    ).toBeVisible();
```
Comment (fangeugene) [inline pwa/tests/team.spec.ts:23]: Why is the sidebar missing events 🤔

File: pwa/tests/team.spec.ts [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+    await expect(
+      page.getByRole('link', { name: 'Sunset Showdown' }),
+    ).toBeVisible();
+    await expect(
+      page.getByRole('link', { name: 'Chezy Champs' }),
+    ).toBeVisible();
+  });
+
+  test('Social media', async ({ page }) => {
```
Comment (fangeugene) [inline pwa/tests/team.spec.ts:50]: watch these tests fail because 604 changes our social media

File: pwa/tests/team.spec.ts [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+    );
+  });
+
+  test('Header', async ({ page }) => {
+    await expect(page.locator('h1')).toHaveText('Team 604 - Quixilver');
+    await expect(
+      page.locator('_react=InlineIcon').nth(0).getByRole('link'),
+    ).toHaveText('San Jose, California, USA');
+    await expect(page.locator('_react=Accordion')).toHaveText(
+      'Leland High School with 15 sponsors',
```
Comment (fangeugene) [inline pwa/tests/team.spec.ts:96]: `with 15 sponsors` is pretty dangerous. it will probably change every year.

File: pwa/tests/team.spec.ts (hunk 1 above)
Comment (tervay) [reply pwa/tests/team.spec.ts:23]: I noticed that too, will resolve later

**Review by fangeugene — APPROVED** [review]

**Pull request merged**